api/v1/database_types.go

```diff
@@ -137,6 +137,8 @@ type (
 		Storages                 map[string]*BackupStorageSpec `json:"storages,omitempty"`
 		Annotations              map[string]string             `json:"annotations,omitempty"`
 		Labels                   map[string]string             `json:"labels,omitempty"`
+		// Type: logical/physical.
```

Comment:
it would be great to elaborate here. Like Type represents a backup type. Can be logical or physical. As another option, we can have BackupType here with two options and we can use them in the validation further.

Reply:
I would also go with the same strategy we have for the EngineType, i.e. create a new type.